src/BikramSambat.ts

```diff
@@ -251,4 +251,16 @@ export default class BikramSambat {
     const month = (this.month + 1) % 12
     return BikramSambat.nepaliMonths[month - 1]
   }
+
+  public getWeekdayNames(language: 'np' | 'en'): string[] {
+    return NepaliDaysData.map((day) => day[language ?? 'np'])
+  }
+
+  public getMonthNamesNepali(language: 'np' | 'en'): string[] {
+    return NepaliMonthsData.map((month) => month[language ?? 'np'])
+  }
+
+  public getMonthNamesEnglish(): string[] {
+    return NepaliMonthsData.map((month) => month.en)
+  }
```

Comment:
these two functions were supposed to be combined in to single `getMonthNames` right?